ui/app/AppLayouts/Communities/panels/MintTokensSettingsPanel.qml

```diff
@@ -257,265 +258,275 @@ StackView {
+            }
 
-                onRemoteDestructRequested: {
-                    remotelyDestructPopup.open()
-                    // TODO: set the address selected in the popup's list
-                }
+            onRemoteDestructRequested: {
+                remotelyDestructPopup.open()
+                // TODO: set the address selected in the popup's list
             }
+        }
 
-            footer: MintTokensFooterPanel {
-                id: footer
+        footer: MintTokensFooterPanel {
+            id: footer
 
-                readonly property TokenObject token: view.token
+            readonly property TokenObject token: view.token
 
-                readonly property bool deployStateCompleted:
-                    token.deployState === Constants.ContractTransactionStatus.Completed
+            readonly property bool deployStateCompleted:
+                token.deployState === Constants.ContractTransactionStatus.Completed
 
-                function closePopups() {
-                    remotelyDestructPopup.close()
-                    alertPopup.close()
-                    signTransactionPopup.close()
-                    burnTokensPopup.close()
-                }
+            function closePopups() {
+                remotelyDestructPopup.close()
+                alertPopup.close()
+                signTransactionPopup.close()
+                burnTokensPopup.close()
+            }
 
-                airdropEnabled: deployStateCompleted &&
-                                (token.infiniteSupply ||
-                                 token.remainingTokens !== 0)
+            airdropEnabled: deployStateCompleted &&
+                            (token.infiniteSupply ||
+                             token.remainingTokens !== 0)
 
-                remotelyDestructEnabled: deployStateCompleted &&
-                                         !!view.tokenOwnersModel &&
-                                         view.tokenOwnersModel.count > 0
+            remotelyDestructEnabled: deployStateCompleted &&
+                                     !!view.tokenOwnersModel &&
+                                     view.tokenOwnersModel.count > 0
 
-                burnEnabled: deployStateCompleted
+            burnEnabled: deployStateCompleted
 
-                remotelyDestructVisible: token.remotelyDestruct
-                burnVisible: !token.infiniteSupply
+            remotelyDestructVisible: token.remotelyDestruct
+            burnVisible: !token.infiniteSupply
 
-                onAirdropClicked:root.airdropToken(view.airdropKey, // tokenKey instead when backend airdrop ready to use key instead of symbol
-                                      view.tokenType, [])
+            onAirdropClicked:root.airdropToken(view.airdropKey, // tokenKey instead when backend airdrop ready to use key instead of symbol
```

Comment:
Is this comment still valid?

Reply:
Yes, yes! Now we are using `symbol` as key role for airdropping. In some point, this will be changed to only `key` role also for airdrop.

Reply:
I think so. I didn't change anything related to `airdropKey` origin.